test/performance/scripts/create-outgoing-payments.js

```diff
@@ -0,0 +1,182 @@
+  // },
+
+  // Uncomment this section to enable the use of Browser API in your tests.
+  //
+  // See https://grafana.com/docs/k6/latest/using-k6-browser/running-browser-tests/ to learn more
+  // about using Browser API in your test scripts.
+  //
+  // scenarios: {
+  //   // The scenario name appears in the result summary, tags, and so on.
+  //   // You can give the scenario any name, as long as each name in the script is unique.
+  //   ui: {
+  //     // Executor is a mandatory parameter for browser-based tests.
+  //     // Shared iterations in this case tells k6 to reuse VUs to execute iterations.
+  //     //
+  //     // See https://grafana.com/docs/k6/latest/using-k6/scenarios/executors/ for other executor types.
+  //     executor: 'shared-iterations',
+  //     options: {
+  //       browser: {
+  //         // This is a mandatory parameter that instructs k6 to launch and
+  //         // connect to a chromium-based browser, and use it to run UI-based
+  //         // tests.
+  //         type: 'chromium',
+  //       },
+  //     },
+  //   },
+  // }
+}
+
+// The function that defines VU logic.
+//
+// See https://grafana.com/docs/k6/latest/examples/get-started-with-k6/ to learn more
+// about authoring k6 scripts.
+//
+export default function () {
+  const headers = {
+    'Content-Type': 'application/json'
+  }
+
+  const CLOUD_NINE_GQL_ENDPOINT =
+    'http://cloud-nine-wallet-backend:3001/graphql'
+  const CLOUD_NINE_WALLET_ADDRESS =
+    'https://cloud-nine-wallet-backend/accounts/gfranklin'
+  const HAPPY_LIFE_BANK_WALLET_ADDRESS =
+    'https://happy-life-bank-backend/accounts/pfry'
+
+  const c9WalletAddressesRes = http.post(
+    CLOUD_NINE_GQL_ENDPOINT,
+    JSON.stringify({
+      query: `
+    query GetWalletAddresses {
+      walletAddresses {
+        edges {
+          node {
+            id
+            url
+          }
+        }
+      }
+    }
+  `
+    }),
+    { headers }
+  )
+  const c9WalletAddresses = JSON.parse(c9WalletAddressesRes.body).data
+    .walletAddresses.edges
+  const c9WalletAddress = c9WalletAddresses.find(
+    (edge) => edge.node.url === CLOUD_NINE_WALLET_ADDRESS
+  ).node
+
+  const createReceiverResponse = http.post(
+    CLOUD_NINE_GQL_ENDPOINT,
+    JSON.stringify({
+      query: `
+        mutation CreateReceiver($input: CreateReceiverInput!) {
+          createReceiver(input: $input) {
+            receiver {
+              id
+            }
+          }
+        }
+      `,
+      variables: {
+        input: {
+          expiresAt: null,
+          metadata: {
+            description: 'Hello my friend',
+            externalRef: null
+          },
+          incomingAmount: {
+            assetCode: 'USD',
+            assetScale: 2,
+            value: 1002
+          },
+          walletAddressUrl: HAPPY_LIFE_BANK_WALLET_ADDRESS
+        }
+      }
+    }),
+    {
+      headers
+    }
+  )
+  const receiver = JSON.parse(createReceiverResponse.body).data.createReceiver
```

Comment:
Im seeing some errors like this when spinning up the localenv and running `pnpm --filter performance run-tests-docker`:

```
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createReceiver' of undefined\n\tat file:///scripts/create-outgoing-payments.js:125:64(84)\n" executor=constant-vus scenario=default source=stacktrace
time="2024-08-05T18:40:26Z" level=error msg="TypeError: Cannot read property 'createQuote' of undefined\n\tat file:///scripts/create-outgoing-payments.js:159:58(128)\n" executor=constant-vus scenario=default source=stacktrace
```

9 TypeErrors for `createReceiver` and then 1 `createQuote`. Looks like its indicating `data` in `JSON.parse(createReceiverResponse.body).data` is `undefined`. 

Im not sure what to make of it though because then the VUs seem to run fine. I see some quote times in http://localhost:4500/ but not sure there is enough info there to make much sense of the errors.

If I run `pnpm --filter performance run-tests-docker` again without restarting the localenv I dont see these TypeErrors.

As a side note, I was surprised to see no outgoing payment based metrics but this is actually consistent with the bruno peer-to-peer example (guess it happens outside of the outgoing payment lifecycle where these are recorded).

Reply:
view of dashboard after running:

![image](https://github.com/user-attachments/assets/87fa270c-a00c-420b-aaa0-14c3063fb443)

Reply:
I think these errors are performance issues being exposed by the test. It seems to work more consistently after [bringing the number of virtual users down to one](https://github.com/interledger/rafiki/pull/2828#discussion_r1704590874), so I suspect that these are the service's way of expressing that it's being overwhelmed.

Reply:
makes sense... I actually dont see it anymore